runtime-modules/content/src/benchmarks/benchmarking.rs

```diff
@@ -1797,6 +1792,642 @@ benchmarks! {
             );
         }
 
+    // ================================================================================
+    // ============================ NFT - BASIC  ======================================
+    // ================================================================================
+
+    // WORST CASE SCENARIO:
+    // STATE COMPLEXITY
+    // - curator owned channel
+    // - curator number is max
+    // - curator has max number of permissions
+    // - channel has all features paused except necessary ones
+    // - channel has max assets
+    // - English Auction with max whitelisted member & some royalty
+    // - nft limits are set
```

Comment:
```suggestion
    // - nft limits are set
    // - video has max. number of assets
```

(should also be the case for all nft benchmarks)